torch/functional.py

```diff
@@ -1429,6 +1430,10 @@ def _lu_impl(A, pivot=True, get_infos=False, out=None):
     .. note::
        ``L``, ``U``, and ``P`` can be derived using :func:`torch.lu_unpack`.
 
+    .. warning::
+        The LU factorization does support backward compatibility,
```

Comment:
I am not sure what this means? Which part is backward compatible?

EDIT: I think you just have an extra "compatibility" here haha